frontend/pages/Tags.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
```

Comment:
Let's relabel this file to Tag or TagView, since we are not using it to render a list of tags, but to show just one tag